Tema OOP/.idea/compiler.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
please add .gitignore to the project.
.idea folder should not be pull requested